src/status_im/utils/number_test.cljs

```diff
@@ -0,0 +1,21 @@
+(ns status-im.utils.number-test
+  (:require [cljs.test :refer-macros [deftest testing are]]
+            [status-im.utils.number :as number]))
+
+(deftest number->nk-test
+  (testing "Bad cases"
```

Comment:
Thank you for tagging me and for unit test! 👍 
1) IMO naming "Positive cases" / "Negative cases" is better 
2) I'd extend positive scenarios:
 - values that should be converted to decimals (i.e. 1100 -> 1.1K)
 - decimals values that should not be converted 0.01; 0 
 3) Negative scenarios with:
  - Exponential format: 1e3
  - Decimal with unexpected delimiter,  i.e. 100,3
  
  Tell me if it doesn't make sense

Reply:
Done 😄